### ISSUE TITLE
Guide to flash firmware

### ISSUE THREAD

**Issue by dzungpv** [gysmo38/mitsubishi2MQTT#4]
Hello, Could you provide instruction how to use the firmware?
Like what is the arduino version, libraries version...

**Comment by gysmo38** [comment]
I user Arduino 1.8.9
Librairy used ArduinoJson version 6.11.3, EspMQTTClient 1.6.0
Flash size use 1M (64K SPIFFS)

**Comment by dzungpv** [comment]
@gysmo38 i failed to build it for the first time and some bug with your code, mqtt password will not set.
i must add .c_string() for some value until it could build, but you use the same Serial port for both debug and read from the heatpump?

**Comment by gysmo38** [comment]
Strange I just build it again today without problem. Do you try with an ESP8266?
I use serial port during boot to show logs and after use it to connect to the heatpump. It is a first stage, I have the idea to put log on a file and show log on webpage.

**Comment by dzungpv** [comment]
Yes, after change some i could build it. Password protect the ui must be a nice feature, like tasmota

**Comment by dzungpv** [comment]
@gysmo38 I ask about the stability, do you have any problem?

**Comment by gysmo38** [comment]
@dzungpv I have 5 units and it work well during more than 2 weeks without problem.

**Comment by dzungpv** [comment]
@gysmo38 why you are not use current status like official example here: https://github.com/gysmo38/mitsubishi2MQTT/blob/0116a6e77b5143a4e5820e146562c1ddc3e21f30/src/mitsubishi2mqtt/mitsubishi2mqtt.ino#L570 and why you remove hp.setStatusChangedCallback(hpStatusChanged) ?

**Comment by gysmo38** [comment]
current status not send all status (just current temperature, set temperature and mode)
I do not find it usefull. It seems to work without it. I prefere use getSettings. 
A new version is comming in few days :)

**Comment by dzungpv** [comment]
@gysmo38 I fixed some problem here: https://github.com/gysmo38/mitsubishi2MQTT/pull/5.
current status will very helpful, i will add hvac_action when hass version 0.98 out in your code. like this: https://github.com/SwiCago/HeatPump/commit/70dfad191b0db3445dc0304c74777e957057690c
